src/Illuminate/Support/Facades/Response.php

```diff
@@ -6,7 +6,7 @@
 
 /**
  * @method static \Illuminate\Http\Response make(string $content = '', int $status = 200, array $headers = [])
- * @method static \Illuminate\Http\Response view(string $view, array $data = [], int $status = 200, array $headers = [])
+ * @method static \Illuminate\Http\Response view(string $view, \Illuminate\Contracts\Support\Arrayable | array $data = [], int $status = 200, array $headers = [])
```

Comment:
There should be no space around the `|`

Reply:
It doesn't matter. Moreover, there are spaces around `|` in other facades too:
- https://github.com/laravel/framework/blob/5.7/src/Illuminate/Support/Facades/Event.php
- https://github.com/laravel/framework/blob/5.7/src/Illuminate/Support/Facades/Gate.php
- https://github.com/laravel/framework/blob/5.7/src/Illuminate/Support/Facades/Lang.php
- https://github.com/laravel/framework/blob/5.7/src/Illuminate/Support/Facades/Validator.php

I suggest you can create a new PR to remove them.

Reply:
Just because there are mistakes in other parts of the framework doesn't mean that it doesn't matter.